src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -223,7 +223,7 @@ class IssueListOverview extends React.Component<Props, State> {
     // If any important url parameter changed or saved search changed
```

Comment:
switch to `isSelectionEqual` here?

Reply:
I was worried that the utc parameter was relevant in this usage as it goes into API payloads.